src/main/java/io/reactivex/plugins/RxJavaPlugins.java

```diff
@@ -377,6 +382,42 @@ public static void onError(@NonNull Throwable error) {
         uncaught(error);
     }
 
+    /**
+     * Checks if the given error is one of the already named
+     * bug cases that should pass through {@link #onError(Throwable)}
+     * as is.
+     * @param error the error to check
+     * @return true if the error should pass throug, false if
+     * it may be wrapped into an UndeliverableException
+     */
+    static boolean isBug(Throwable error) {
```

Comment:
Shouldn't `CompositeException` be in here as well? While technically they're _always_ undeliverable, they're very crash-worthy exceptions since they indicate an error while processing an error.